app/models/friendship.rb

```diff
@@ -1,4 +1,40 @@
 class Friendship < ActiveRecord::Base
+  after_initialize :set_defaults, unless: :persisted?
+  after_create :create_inverse, if: :inverse_record_nil?
+  after_update :update_inverse
+  after_destroy :destroy_inverse
+
   belongs_to :user
   belongs_to :friend, class_name: 'User'
+
+  def set_defaults
+    self.confirmed = false if confirmed.nil?
+  end
+
+  def create_inverse
```

Comment:
- [ ] Thanks to this method, we can find user friends only by user_id in friendships table. 
We can now use SQL queries instead of ruby methods to interact with the DB.